### PR TITLE
feat(getting-started/setting-up-flynt): add next steps section 

### LINKED ISSUE #51: link to some next steps in getting-started/setting-up-flynt.md
I feel I'm a little stuck on this page. As this might be one of my very first entry points to getting my hands on Flynt, it could be beneficial to link to some next steps at the end of the page. Maybe even to the "Develop a Simple Flynt Component in 6 Steps" Tutorial?

### PR DESCRIPTION
**Pull request by emcarru**
...and combine it with the previous "Need Help?" alert.

resolves #51

### CODE REVIEW

File: content/guide/getting-started/setting-up-flynt.md [hunk 1]
```diff
@@ -52,11 +52,10 @@ As soon as you have your environment set up, follow the steps below to create a
...
-    <li><a href="https://github.com/flyntwp/flynt-starter-theme/issues">Github - Starter Theme Issues</a></li>
-  </ul>
-</div>
+## Resources / Next Steps
+
+1. Learn how to [develop your first simple Flynt component in 6 steps](https://flyntwp.com/develop-a-simple-flynt-component-in-6-steps/).
+2. Get an understanding of the [Flynt Ecosystem](https://flyntwp.com/getting-to-know-the-flynt-ecosystem/).
+3. If you are stuck or having issues setting up Flynt, leave an issue on Github for further assistance:
+  - [Github - Core Plugin Issues](https://github.com/flyntwp/flynt-core/issues)
+  - [Github - Starter Theme Issues](https://github.com/flyntwp/flynt-starter-theme/issues)
```
Comment (Qakulukiam) [inline content/guide/getting-started/setting-up-flynt.md]: isn't the spelling GitHub?

Comment (emcarru) [inline content/guide/getting-started/setting-up-flynt.md]: yes, thanks

File: content/guide/getting-started/setting-up-flynt.md [hunk 2]
```diff
@@ -57,5 +57,5 @@ As soon as you have your environment set up, follow the steps below to create a
 1. Learn how to [develop your first simple Flynt component in 6 steps](https://flyntwp.com/develop-a-simple-flynt-component-in-6-steps/).
 2. Get an understanding of the [Flynt Ecosystem](https://flyntwp.com/getting-to-know-the-flynt-ecosystem/).
 3. If you are stuck or having issues setting up Flynt, leave an issue on Github for further assistance:
```
Comment (Qakulukiam) [inline content/guide/getting-started/setting-up-flynt.md]: missed one

Comment (emcarru) [inline content/guide/getting-started/setting-up-flynt.md]: Gosh darn

**Pull request merged**